common/types/offer.go

```diff
@@ -68,9 +68,9 @@ func (o *Offer) IsSet() bool {
 
 // OfferExtra represents extra data that is passed when an offer is made.
 type OfferExtra struct {
-	StatusCh          chan Status
-	RelayerEndpoint   string
-	RelayerCommission float64
+	StatusCh          chan Status `json:"-"`
+	RelayerEndpoint   string      `json:"relayerEndpoint"`
+	RelayerCommission float64     `json:"relayerCommission"`
```

Comment:
Is the `RelayerCommission` used anywhere that a slight difference in the bits that are written out during serialization and the bits that are read back in could cause an issue?

Reply:
I want to say probably not but I'd have to double check how much precision is lost. `RelayerCommission` will generally be on the order of `0.01`, do you think that's acceptable?

Reply:
As long as we are not taking a hash of the result anywhere, a slight difference in the binary representation probably doesn't matter. Here's the example I was seeing somewhere else in our code with an exchange rate of 0.05 being reprinted out. (Not a binary difference, but a string difference in that example.)
```
func TestFloat(t *testing.T) {
	jsonData := []byte("0.05")
	var f float64
	err := json.Unmarshal(jsonData, &f)
	require.NoError(t, err)
	t.Logf("%.20f", f)
}
```
Output:
```
0.05000000000000000278
```

Reply:
good point, this isn't getting hashed anywhere so I think it should be okay. but definitely something to keep in mind